src/lib.rs

```diff
@@ -20,14 +22,21 @@ pub fn get_lang() -> Option<String> {
     web_sys::window().and_then(|w| w.navigator().language())
 }
 
-//TODO: support linux
 #[cfg(target_os = "linux")]
 pub fn get_lang() -> Option<String> {
-    None
+    use std::env;
+
+    if let Ok(value) = env::var("LC_ALL")
```

Comment:
Is there no api on Linux? lol is it just in various env variables?

Reply:
AFAIK this is the way that provides the most coverage for various different desktop environments. One thing that I changed now is add code to trim the end of the variable since it contains [codeset](https://wiki.archlinux.org/title/Locale) which is not so useful here.